DOC.md

```diff
@@ -1094,6 +1094,47 @@ To use any `*SELECT*` variable, the `store_selection_keys` must be set via
 hitting `<Tab>` while in Visualmode will populate the `*SELECT*`-vars for the next
 snippet and then clear them.
 
+## Environemt Namespaces
+
+You can also add your own variables by using the `ls.env_namespace(name, opts)` where:
+
+* `name`  is string that does not contains the character "_" and names the namespace.
+* `opts` is a table containing:
+    * `vars` 
```

Comment:
How about stating the type of these arguments explicitly (+give function-prototype, for functions)? I think that might make it much easier to grasp at a glance

Reply:
Not sure what you mean by explicitly, type signatures? if is that the case what format should we use?

Reply:
Yeah, type signatures :+1:
So far I've used `fn(arg1name: arg1type, arg2name: arg2type) -> returnType`, different possiblities as `(type1|type2|type3)` (I _think_ I'm consistent with that one)